src/app/containers/App/App.jsx

```diff
@@ -66,7 +66,17 @@ export const App = ({ routes, location, initialData, bbcOrigin }) => {
     }
   }, [routes, location.pathname]);
 
-  return renderRoutes(routes, { ...state, bbcOrigin });
+  const pathRef = useRef(null);
```

Comment:
Can these changes be extracted into their own file and tested in isolation please, ideally though mounting and updating the props. Could this use a generic `usePrevious` hook thats in a util folder?